plugin/utils/flag.py

```diff
@@ -0,0 +1,33 @@
+class Flag:
+    """ utility class for storing possibly separated flag """
```

Comment:
nit: Should have no spaces within quotes?

Sorry about pointing out such details but you seem to be fixing some places to be consistent and then introducing "wrong" style in new code so I was wondering if that's on purpose. ;)